.config/dotnet-tools.json

```diff
@@ -6,7 +6,15 @@
       "version": "20.0.0",
       "commands": [
         "fsdocs"
-      ]
+      ],
+      "rollForward": false
+    },
+    "fsharp-analyzers": {
+      "version": "0.32.0",
+      "commands": [
+        "fsharp-analyzers"
+      ],
```

Comment:
For testing this feature out, I had to add this. It would probably be better to have some properties we can override for local dev, similar to how [Myriad does it](https://github.com/MoiraeSoftware/Myriad/blob/680546237bf5a718eda8593a507d9ee2b7771da4/test/Myriad.IntegrationPluginTests/Myriad.IntegrationPluginTests.fsproj#L67-L68)